docs/reference/ilm/actions/ilm-searchable-snapshot.asciidoc

```diff
@@ -31,6 +31,10 @@ Using a policy that makes use of the <<ilm-rollover, rollover>> action
 in the hot phase will avoid this situation and the need for a manual rollover for future
 managed indices.
 
+[NOTE]
+Migration from the warm to the cold tier requires snapshotting the data in the repo
+and reading it back to the node which incurs data transfer costs.
```

Comment:
Thanks for this update @arteam!

Is this isolated to the warm->cold transition?

It seems like you'd incur this cost whenever you create a searchable snapshot, regardless of the tier or phase transition. For example, if someone goes directly from hot->frozen, it seems like you'd incur the same data transfer costs.

Reply:
I think we should elaborate a bit here, mentioning that this is also true when hot/warm and cold are co-located. I think this is the part that can confuse some users. Also, we should note that if you already take snapshots of your data, the actual data uploaded to the repo will be minimal. Finally, I am a bit worried about the terminology "data transfer costs", I think we should use clarify that it requires a download of data which may have costs depending on operating environment. Perhaps link to what David wrote in #77607 rather than elaborate too much here on the costs.

I think `frozen` is less confusing since:
1. We recommend using dedicated frozen nodes.
2. Partially mounted indices will only download a minimum set of data initially.
3. The bulk of the download happens on search.

Still, adding a note may be worthwhile

Reply:
I think #77607 kind of supersedes this change and makes it superfluous. #77607 explains in detail all the points about the migration between the tiers   including the structure of "data transfer costs".